ansible/files/archives/monitoring/opt/zabbix/etc/scripts/vm-cpu-monitor

```diff
@@ -79,7 +79,7 @@ while(my $line = <PRSTAT>) {
 
 		foreach $zonename (keys %cpu_stats) {
 			# Skip OS zones for now	
-			next if $cpu_stats{$zonename}->{'type'} eq 'OS';
+			next if $cpu_stats{$zonename}->{'type'} ne 'KVM';
```

Comment:
Please add also `bhyve` type so we don't have to modify it again in next release.
Otherwise LGTM.
Thx.

Reply:
@YanChii With the condition like this, `BHYVE` should be also excluded, which is what we want, or don't we?

Reply:
Maybe I don't quite understand the purpose of this filter. I just know that currently there are 4 brands: `KVM`, `bhyve`, `OS` and `LX`. 

BTW Joyent introduced the `"hvm": true` json parameter to identify both `KVM` and `bhyve`.